spring-cql/src/test/java/org/springframework/cassandra/core/OrderingTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cassandra.core;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.assertj.core.api.AssertionsForInterfaceTypes;
+import org.junit.Test;
+
+/**
+ * Unit tests for class {@link Ordering org.springframework.cassandra.core.Ordering}.
+ *
+ * @author Michael Hausegger, hausegger.michael@googlemail.com
+ */
+public class OrderingTest {
+
+
+	@Test
+	public void testCompareWithNonNullReturningPositive() throws Exception {
```

Comment:
`test` prefix in methods isn't required since JUnit 4. Please express the goal of the test in the method name (such as `ascendingComparesLessThanToDescending`)